### ISSUE TITLE
[WotLK] Starting Items Invalid

### ISSUE THREAD

**Issue by Enverex** [unified-db/Database#201]
I've not seen any other posts about this so I'm not sure if I have some sort of weird isolated issue, but here it is:

Whenever someone creates a new character on my server, they start with invalid items. The items look correct on the character paper-doll and in the character slots, but if you drag them to your inventory then they just show with a Question Mark item. They also do not fulfill the criteria for their skills, e.g. the invalid axe that a character may start isn't classed as a melee weapon.

All other items dropped or found appear to be fine, it's just the starting items.

**Comment by Grz3s** [comment]
hmm.. strange...
I just tested fresh Dwarf Warrior -- all items moved into inventory just fine...
Pls give me more details ... 
what race, class ... 
Ill check again...but it looks so far that its your server problem :)

**Comment by Enverex** [comment]
Here they are both on the paper-doll (as they come initially) and then after I've moved them to the characters inventory.

Seems to happen to any race/class. Tried a few at random, all had the same issue. They seem to be given the correct equiptment (as you can see from the first picture) but it doesn't work.

![character](https://cloud.githubusercontent.com/assets/958670/14943402/207ff03e-0fd0-11e6-919c-ea5e7fdee137.png)

![inventory](https://cloud.githubusercontent.com/assets/958670/14943403/2093dcca-0fd0-11e6-8d95-cf58a4267be5.png)

**Comment by Enverex** [comment]
Also, to clarify, I've re-ran the database populator which nuked everything in the main mangos DB and added all the items back in and then tested that, same behaviour.

**Comment by Grz3s** [comment]
ok all info is needed then:
- do you run newest fresh cmangos core?
- is DB a fresh [478] UDB?
- ACID and sd2 added correctly?
- any errors in DBErrors.log ?

**Comment by Enverex** [comment]
Yes, everything was pulled from GIT a few days ago at most (with the exception of UDB which is from earlier today). I installed the database via the InstallFullDB script (with the CORE and ACID settings filled in). Nothing odd in the DB log.

**Comment by Grz3s** [comment]
hmm...
pls recheck your 'characters' DB if is up to date.
You should see in 'character_db_version'  = required_12937_01_characters_pvpstats

**Comment by Enverex** [comment]
required_12937_01_characters_pvpstats - Seems to match.

**Comment by cyberium** [comment]
Try delete in your wow folder "cache/wdb" folder

**Comment by Enverex** [comment]
This has happened to multiple people on different machines, so it shouldn't be local client cache related. I can try that when I'm back at that machine, but the above point makes it seem like it's not going to be related.

**Comment by cyberium** [comment]
If no one is able to find a way to reproduce it, it will be hard to fix anything on this.

**Comment by bsonntag** [comment]
This is happening to me, too.
Fresh install, `git clone`d everything just four days ago. I also ran InstallFullDB.sh with correct CORE and ACID settings.
Don't know if it's important, but I'm installing cmangos on Linux Mint.

**Comment by Grz3s** [comment]
seriously I don't think ..this is a DB issue...
It may be smth with your mangos.conf settings or connection problems....

**Comment by Grz3s** [comment]
noone complain about this ... after 2016..
closed
